nginx/conf.d/default.conf

```diff
@@ -0,0 +1,21 @@
+server {
+    listen 5000 default;
+    server_name _;
+    server_tokens off;
+
+    if ($http_x_forwarded_proto = "http") {
+        return 301 https://$host$request_uri;
+    }
+
+    location /status {
+        default_type application/json;
+        return 200 '{"status": "Unavailable"}';
```

Comment:
Does it need to send `json` response?

Reply:
Runner returns the application version on this route. The Runner uptime checks use this endpoint. There is an argument that when the holding page is up we should be generating uptime failure (which I'd considered). However, my thoughts are; the deployment of the maintenance page will be a controlled and deliberate action so we don't need notification that Runner is down. But more importantly, we need upcheck on the maintenance page itself and continuing to use the `/status` endpoint and the existing notification makes sense.